Issue #2983 #fixed link (#3022)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfbOnline/set-up-skype-for-business-online/troubleshoot-problems-with-skype-for-business-online.md

## Troubleshoot admin issues for Skype for Business Online

> You can get answers for most of your Skype for Business Online questions by [Check for common causes of Skype for Business Online sign-in errors](troubleshooting-sign-in-errors-for-admins.md#toc323194094). This KB article also includes many troubleshooting resources and error code references: [How to troubleshoot sign-in issues in Skype for Business Online](https://docs.microsoft.com/skypeforbusiness/set-up-skype-for-business-online/troubleshooting-sign-in-errors-for-admins).

Reviewer: ```suggestion
You can get answers for most of your Skype for Business Online questions in the article [Check for common causes of Skype for Business Online sign-in errors](troubleshooting-sign-in-errors-for-admins.md#toc323194094). The following KB article also includes many troubleshooting resources and error code references: [Troubleshooting Skype for Business Online sign-in errors for administrators](https://docs.microsoft.com/skypeforbusiness/set-up-skype-for-business-online/troubleshooting-sign-in-errors-for-admins).
```
